docs/source/creating_databases.rst

```diff
@@ -0,0 +1,91 @@
+Creating the databases
+======================
+
+First, make sure you have Python and Postgres installed:
+
+-  :doc:`installing_python`
+-  :doc:`installing_postgres`
+
+
+Creating the databases
+----------------------
+
+OSX
+~~~
+
+After installing Postgres, you will need to create two databases:
+one for your experiments to use, and a second to support importing saved
+experiments. It is recommended that you also create a database user.
+
+Naviagate to a terminal and type:
+::
+
+    createuser -P dallinger --createdb
+    (Password: dallinger)
+    createdb -O dallinger dallinger
+    createdb -O dallinger dallinger-import
+
+
+The first command will create a user named ``dallinger`` and prompt you for a
+password. The second and third command will create the ``dallinger`` and 
+``dallinger-import`` databases, setting the newly created user as the owner.
+
+You can optionally inspect your databases by entering ``psql``. 
+Inside psql you can use commands to see the roles and database tables:
+::
+
+    \du
+    \l
+
+To quit:
+::
+
+    \q
+
+If you get a "psql: FATAL: database "xxxxx" does not exist" error, where "xxxxx" is likely
```

Comment:
If you have them run `psql dallinger`, you could remove quite a bit here. I think I'd still include something like:

> "If you get a “psql: FATAL: database “xxxxx” does not exist” error, where “xxxxx” is likely the name of your OSX username, you most likely forgot to specify the `dallinger` database when running `psql`. The command should include the database name: `psql dallinger`."

Reply:
:+1:

Reply:
Tested to be working